extensions/ui/org.eclipse.smarthome.ui.paper/about.html

```diff
@@ -80,6 +80,21 @@
 		<a href="https://github.com/choffmeister/roboto-fontface-bower">github/</a> and are made available under the 
 		<a href="about_files/ROBOTO_FONT_LICENSE">Apache License</a>. 
 		</p>
+		
+		<p>
+        <strong>JQueryUI</strong> <br/><br/>
```

Comment:
That is because about.html was excluded from formatting pull request. So I also did not format it.